spec/spec_helper.rb

```diff
@@ -1,6 +1,9 @@
 require 'capybara/poltergeist'
 
 Capybara.javascript_driver = :poltergeist
+Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(app, {debug: true, default_wait_time: 30, timeout: 90})
```

Comment:
Line is too long. [91/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.